indra/llmath/llmath.h

```diff
@@ -536,7 +536,8 @@ inline void ll_remove_outliers(std::vector<VEC_TYPE>& data, F32 k)
 // Note: in our code, values labeled as sRGB are ALWAYS gamma corrected linear values, NOT linear values with monitor gamma applied
 // Note: stored color values should always be gamma corrected linear (i.e. the values returned from an on-screen color swatch)
 // Note: DO NOT cache the conversion.  This leads to error prone synchronization and is actually slower in the typical case due to cache misses
-inline float linearTosRGB(const float val) {
+inline float linearTosRGB(const float val)
```

Comment:
Did you intend to change `linearTosRGB()` and `sRGBtoLinear()` to `constexpr` as well?

Reply:
They both call `pow` which isn't constexpr yet (starting with C++26)

Reply:
Aha. Maybe a comment mentioning that in each place? I can imagine myself thinking, "Hmm, looks like someone forgot to put `constexpr` here," and making the change, and getting the error, and fixing it back.